app/views/foreman_tasks/api/tasks/show.json.rabl

```diff
@@ -1,5 +1,5 @@
 object @task if @task
 
-attributes :id, :label, :pending
+attributes :id, :label, :pending, :action
```

Comment:
I followed the `task_label` from:
https://github.com/theforeman/foreman-tasks/blob/master/app/views/foreman_tasks/tasks/_details.html.erb#L92

It seems the end result is the action field so the new strategy is to add it to the json result.

Reply:
Just to have complete picture, @adamruzicka what is the difference between the `action` and `humanized.action` fields, please?

Reply:
Each action can define a human readable name, this is `humanized.action` and is defined on an action class as `#humanized_name`.  Just `action` is generated from various parts of `humanized`.

For example for `Actions::RemoteExecution::RunHostJob` action:

```ruby
pry(main)> task.humanized[:action]
=> "Remote action:"
pry(main)> task.humanized
=> {:action=>"Remote action:", :input=>"Run true on stub-100", :output=>"»Exit status: 0«", :errors=>[]}
pry(main)> task.action
=> "Remote action: Run true on stub-100"
```

Reply:
Got it, thanks.